standards/SPIFFE_Workload_API.md

```diff
@@ -176,21 +182,6 @@ message X509BundlesResponse {
     // Bundles are ASN.1 DER encoded.
     map<string, bytes> bundles = 2;
 }
-
```

Comment:
Good catch! Thanks.

Reply:
Heh, I almost just pushed a PR for this myself. I'm glad I double checked the open PRs first :)